privval/signer_requestHandler.go

```diff
@@ -81,6 +81,7 @@ func DefaultValidationRequestHandler(
 	case *privvalproto.Message_PingRequest:
 		err, res = nil, mustWrapMsg(&privvalproto.PingResponse{})
 
+	case nil:
```

Comment:
so the signer is sending us `nil` messages? what for?

Reply:
Yeah, this is #5371. I don't really understand why we're setting a read deadline at all, we should just keep waiting until the next request arrives.

Reply:
I'm not sure this is the fix. When testing did you observe ping requests come through or was it always nil?

Reply:
Actual messages were processed too, but I didn't look for pings or anything. I haven't debugged this to any extent at all, I just noticed it while working on something else.

Reply:
> I don't really understand why we're setting a read deadline at all, we should just keep waiting until the next request arrives.

+1. normally you'd use read deadline to drop the bad sender and/or unblock other processes, but we have neither of these things here.

Reply:
Yeah. It would make sense to have a read deadline when waiting for the _response_, but not when waiting for the next request.